src/test/java/edu/kis/powp/jobs2d/command/factory/Factory.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.powp.jobs2d.command.factory;
+
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.command.ComplexCommand;
+import edu.kis.powp.command.OperateToCommand;
+import edu.kis.powp.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.drivers.adapter.DrawingAdapter;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDrawerAdapter;
+
+public class Factory {
+    public static ComplexCommand SquareShape(int x, int y){
+        ComplexCommand figure = new ComplexCommand();
+        DrawingAdapter lineDrawer = new DrawingAdapter();
```

Comment:
Fixed driver instance. Commands should be able to use the current driver (i.e., DriverManager.getCurrentDriver())